flopy/mf6/data/dfn/gwf-csub.dfn

```diff
@@ -1,4 +1,5 @@
 # --------------------- gwf csub options ---------------------
+# package-type advanced-stress-package
```

Comment:
CSUB is not a stress package.

Reply:
removed "stress-package" from CSUB package dfn